wforce/wforce-lua.cc

```diff
@@ -1002,21 +1002,37 @@ vector<std::function<void(void)>> setupLua(bool client, bool multi_lua, LuaConte
     c_lua.writeFunction("showVersion", []() { });
   }
 
+  if (!multi_lua) {
+    c_lua.writeFunction("setKey", [](const std::string& key) {
```

Comment:
I'd prefer if this function returned a status, indicating success/failure.

Reply:
sure